packages/hash/api/src/model/ontology/util.ts

```diff
@@ -0,0 +1,33 @@
+import { WORKSPACE_ACCOUNT_SHORTNAME } from "@hashintel/hash-backend-utils/system";
+import { OrgModel, UserModel } from "..";
+import { GraphApi } from "../../graph";
+import { workspaceAccountId } from "../util";
+
+/**
+ * Get the namespace of an account owner by its id
+ *
+ * @param params.ownerId - the id of the owner
```

Comment:
nit:
From what i can tell the long em-dash is put in after the param in JS docs without explicitly adding this dash (if you remove it and mouse over `getNamespaceOfAccountOwner` it still shows an em-dash), should we just omit the dash or have it in all jsdocs? This isn't a very important question, as the behavior is the same, but for consistency we should probably choose one over the other

Reply:
Ideally I think we should use the dashes, because that's what JSDoc does in [their examples](https://jsdoc.app/howto-es2015-classes.html). I also prefer this because it makes the documentation easier to read in places like the GitHub UI why there's no JSDoc syntax highlighting.

Should I create a task to make our JSDocs consistent for the model classes?

Reply:
I don't feel strongly about this, you can create a task but probably very low priority to get done, and I assume we have the inconsistency across the codebase

Reply:
Yeah, maybe let's try to consistently use the dash with any new JSDoc param definitions we introduce, but otherwise not worry about it too much until we get around to the task. Have created it [here](https://app.asana.com/0/1202805690238892/1203063463721795/f)